docs/guide/piece-pages.md

```diff
@@ -171,7 +171,39 @@ Apostrophe's pager macro adds basic, unstyled pagination to view more. The pager
 
 Show pages are the web pages for individual pieces, rendered from `show.html` templates. Instead of `data.page`, the template uses `data.piece` to access the piece data.
 
-Assuming our `article` piece type example has a single `main` area, it could look like this:
+Assuming our `article` piece type example has a single `body` area, it could look like this:
```

Comment:
```suggestion
Assuming that our `article` piece type example has a single `body` area, it could look like this:
```

not sure, to be checked with a native English speaker

Reply:
I think that `Assuming our` is good.

Reply:
"Assuming our" is fine. Both are acceptable grammar, but "assuming our" is shorter while still being grammatically acceptable.